estuary-cdk/estuary_cdk/capture/__init__.py

```diff
@@ -10,6 +10,7 @@
 import tempfile
 import traceback
 import xxhash
+import orjson
```

Comment:
duplicate import.

These look to be cosmetic -- remove this file from changeset?